serde_json_path/CHANGELOG.md

```diff
@@ -236,7 +236,7 @@ This split was done to accommodate the new `#[function]` attribute macro, which
 
 # 0.5.3 (14 March 2023)
 
-- **fixed:** Fix serialization behaviour of `NodeList` ([#30])
+- **fixed:** Fix serialization behavior of `NodeList` ([#30])
```

Comment:
Not sure if you prefer British or American English spelling; but at least `CODE_OF_CONDUCT.md` was consistently using "behavior".

Reply:
I am Canadian, so I tend to use the British spelling, but I am impartial and will accept this 😄